hadoop-ozone/integration-test/src/test/java/org/apache/hadoop/ozone/client/rpc/TestContainerStateMachineInt.java

```diff
@@ -53,7 +53,7 @@
  * Tests the containerStateMachine failure handling.
  */
 
-public class TestContainerStateMachine {
+public class TestContainerStateMachineInt {
```

Comment:
Why is this change done. Any reason for this?

Reply:
@bharatviswa504 I just changed this since I added a class called TestContainerStateMachine in correct package. So I added "int" to this class name to mark it as an integration test. Do you feel it is OK to have 2 classes with same name?